setup.py

```diff
@@ -715,18 +717,19 @@ def make_relative_rpath(path):
 cmdclass.update(build_dep_cmds)
 
 
-setup(name="torch", version=version,
-      description="Tensors and Dynamic neural networks in Python with strong GPU acceleration",
-      ext_modules=extensions,
-      cmdclass=cmdclass,
-      packages=packages,
-      package_data={'torch': [
-          'lib/*.so*', 'lib/*.dylib*', 'lib/*.dll', 'lib/*.lib',
-          'lib/torch_shm_manager',
-          'lib/*.h',
-          'lib/include/TH/*.h', 'lib/include/TH/generic/*.h',
-          'lib/include/THC/*.h', 'lib/include/THC/generic/*.h',
-          'lib/include/ATen/*.h',
-      ]},
-      install_requires=['pyyaml', 'numpy'],
-      )
+if __name__ == '__main__':
```

Comment:
Why the `if`? I don't think we ever want to import this file anywhere

Reply:
as he mentioned, it's to debug. if we want to know variable values in setup.py, we can import it in an interpreter and inspect (in that case the build is not run).